src/readbin/byteweight_main.ml

```diff
@@ -108,11 +108,12 @@ let dump info length threshold path (input : string) : unit t =
               let new_fs_s = BW.find bw ~length ~threshold mem in
               Addr.Set.union fs_s @@ Addr.Set.of_list new_fs_s
             else fs_s) in
-    Symbols.write_addrset stdout fs_set;
+    Symbols.write_addrs stdout @@ Addr.Set.to_list fs_set;
```

Comment:
A general piece of advice about core associative  containers. To create, use `Key.Container.constructor`, to access use `Container.accessor`, example:

Creation
```
  Addr.Set.empty
  String.Table.create ()
  Exp.Hash_set.create ()
  Stmt.Map.of_list
```

Accessing already created container:
```
   Set.to_list
   Table.find
   Hash_set.add
   Hashtbl.mem
```

Note that `Table` transforms to `Hashtbl`

There is no need to fix it right now, but in general it is nicer, more general and requires you to write less code.